### PR TITLE
add namespaceOverride support

### PR DESCRIPTION
**Pull request by RuStyC0der**
PR regarding issue #2050

### CODE REVIEW

File: charts/aws-ebs-csi-driver/values.yaml
Comment (AndrewSirenko) [inline charts/aws-ebs-csi-driver/values.yaml:334]: Nit: Move this comment and parameter between lines 407-408 because it is less important than these top parameters.

Comment (RuStyC0der) [reply charts/aws-ebs-csi-driver/values.yaml]: Hi, just moved this comment down, also defaulted namespaceOverride to blank string to match values file style

File: charts/aws-ebs-csi-driver/values.yaml
Comment (torredil) [inline charts/aws-ebs-csi-driver/values.yaml:405]: np: `deloy` -> `deploy`

File: charts/aws-ebs-csi-driver/values.yaml
Comment (torredil) [inline charts/aws-ebs-csi-driver/values.yaml:406]: Suggest simplifying this parameter to `namespace`. The `Override` postfix does not add helpful context.

**Review by torredil — COMMENTED** [review]
Thank you for the contribution!

/lgtm

**Review by AndrewSirenko — COMMENTED** [review]
Will lgtm after nits. Thank you!

File: charts/aws-ebs-csi-driver/values.yaml
Comment (RuStyC0der) [reply charts/aws-ebs-csi-driver/values.yaml:406]: I did named it `namespaceOverride` because it overrides namespace that you may set with `--namespace` helm flag. Also I saw it named like that in bunch of other charts so it matches common style but, if you insist we can change it to `namespace`

File: charts/aws-ebs-csi-driver/values.yaml
Comment (RuStyC0der) [reply charts/aws-ebs-csi-driver/values.yaml]: done)

File: charts/aws-ebs-csi-driver/values.yaml
Comment (RuStyC0der) [reply charts/aws-ebs-csi-driver/values.yaml]: done)

File: charts/aws-ebs-csi-driver/values.yaml
Comment (torredil) [reply charts/aws-ebs-csi-driver/values.yaml:405]: That makes sense, `namespaceOverride` works

**Review by ConnorJC3 — CHANGES_REQUESTED** [review]
Please also add the changes from `_node.tpl` to `_node-windows.tpl` - Otherwise lgtm

Edit: Also, you'll have to rebase your PR after we fix the govulncheck issue, will let you know when that's ready

File: charts/aws-ebs-csi-driver/values.yaml
Comment (RuStyC0der) [reply charts/aws-ebs-csi-driver/values.yaml:405]: Good, marking this conversation as resolved)

**Comment by rdpsin** [comment]
Can you squash your commits into one? Thanks.

**Comment by AndrewSirenko** [comment]
/label tide/merge-method-squash

> Can you squash your commits into one?

Prow got some upgrades @rdpsin  😄

**Comment by ConnorJC3** [comment]
Unfortunately, you will need to rebase on top of the latest master to make `govulncheck` happy - once you do that we can merge this in.

**Comment by RuStyC0der** [comment]
Hi @ConnorJC3, just rebased master commits)

**Comment by RuStyC0der** [comment]
Hi, if there is no more issues, shall we get this PR merged?

**Comment by RuStyC0der** [comment]
Thank all you guys for reviewing this PR)  Hope my small enhancement will be useful)

**Pull request merged**
